src/cache.js

```diff
@@ -263,8 +264,13 @@ class Cache {
    * as cleared so that subsequent get operation will not actually return any data cached in persistent storage
    */
   clear() {
-      this._cache = {};
-      this._saveLastCleared();
+    this._cache = {};
+    this._saveLastCleared();
+  }
+
+  remove(key) {
```

Comment:
All public method needs a comment in jsdoc format explaining what it does + description of parameters